src/cheri/cheri_isa_tables.adoc

```diff
@@ -51,6 +51,18 @@ include::generated/{rvy_zicsr_mod_file_name}.adoc[]
 include::generated/{rvy_sentry_insn_file_name}.adoc[]
 |==============================================================================
 
+[#rvy_bld_insn_table, reftext="{rvy_bld_insn_ext_name}"]
+=== {rvy_bld_insn_ext_name}
+
+{rvy_bld_insn_ext_name} adds the {CBLD} instruction.
```

Comment:
Do we need some rationale here? Something like: "This instruction is useful to speed up operations such as paging in memory after swap."?

Reply:
any such description should have something about embedded cores too - so maybe just focus on authorising caps? not sure